src/dbnode/storage/index.go

```diff
@@ -1754,9 +1788,22 @@ func newNamespaceIndexMetrics(
 	opts index.Options,
 	iopts instrument.Options,
 ) nsIndexMetrics {
+	const (
+		indexAttemptName = "index-attempt"
+		forwardIndexName = "forward-index"
+	)
```

Comment:
Does it matter if you define the const here or outside of the function?

Reply:
It does not but this is nice that it's closer to the function in use, I really only do this with metric names.